### PR TITLE
Rivercity Innovations Pull Request

### PR DESCRIPTION
**Pull request by RileyRCI**
#### Summary

...

#### Changes

Added Rivercity Innovations & 2 of our products to the Repo
- ...
- ...

#### Notes for Reviewers

...

#### Release Notes

- ...

### CODE REVIEW

**Comment by Jaime-Trinidad** [comment]
@RileyRCI please sign the CLA

**Comment by Jaime-Trinidad** [comment]
@RileyRCI amazing image of the dog, but as this will be deploy also in the new device repository page, can you change for an image of only the sensor without background, the one of the dog you can leave as second image

**Comment by Jaime-Trinidad** [comment]
> @RileyRCI amazing image of the dog, but as this will be deploy also in the new device repository page, can you change for an image of only the sensor without background, the one of the dog you can leave as second image

@RileyRCI I already see there are the images, thanks

File: vendor/rivercity-innovations/lynx-codec.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+# For documentation on writing encoders and decoders, see: https://thethingsstack.io/integrations/payload-formatters/javascript/
+uplinkDecoder:
+  fileName: lynx.js
+  # Examples (optional)
+  examples:
+    - description: Regular Uplink
+      input:
+        fPort: 2
+        bytes: [0x03, 0x1B, 0xA1, 0x77, 0xF9, 0xA4, 0xFE, 0xF1, 0x94, 0xDF, 0x26]
+      output:
```
Comment (Jaime-Trinidad) [inline vendor/rivercity-innovations/lynx-codec.yaml:11]: Please review the input is not matching the output:

`"data":{"batV":4,"direction":314.8235294117647,"fixFailed":false,"inTrip":true,"latitudeDeg":52.142455,"longitudeDeg":-106.627343,"temperature":19,"type":"position"}}`
 does not match 
`{"data":{"latitudeDeg":52.142455,"longitudeDeg": -106.627343,"inTrip":true,"fixFailed":false,"batV":4,"direction":314.8,"temperature":19}}`

File: vendor/rivercity-innovations/txh-codec.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+          TemperatureLimitLow: 0.0
+          HumidityLimitHigh: 90.0
+          HumidityLimitLow: 10.0
+          DataAverageNumber: 3
+          TemperatureAlertsEnabled: true
+          HumidityAlertsEnabled: false
+          AdrOn: true 
+          ConfirmedUplinks: true
+          IsAlertsConfirmed: true
+    - description: Uplink and Check times
```
Comment (Jaime-Trinidad) [inline vendor/rivercity-innovations/txh-codec.yaml:49]: Also review this example, output does not match:
`output {"data":{"CheckTimeSeconds":60,"ReportsNumChecksAlert":20,"ReportsNumChecksRegular":30,"bytes":[0,0,0,60,0,30,0,20]}}`
 does not match 
`{"data":{"CheckTimeSeconds":60,"ReportsNumChecksRegular":30,"ReportsNumChecksAlert":20}}`

**Comment by RileyRCI** [comment]
Made codec & .js file fixes. Fixed improper photo format. Validation passed successfully, waiting for review.

**Review by Jaime-Trinidad — APPROVED** [review]

**Pull request merged**